GetF!t/Shared/Styles and buttons/ModalSheetStyle.swift

```diff
@@ -0,0 +1,59 @@
+import SwiftUI
+
+struct ModalSheetStyle: View {
```

Comment:
premenovala by som na ModalSheetView

Reply:
Celkovo by som to asi trochu prerobila 🙂 . Ak sa to bude volať ModalSheetView tak by som od toho čakala, že mu dáš nejaký content a on zobrazí content v tom modále. Takže by som pridala atribút content a použila by som ho vo body 🙂